package/MDAnalysis/core/universe.py

```diff
@@ -1289,6 +1289,41 @@ def _fragdict(self):
                 fragdict[a.ix] = fraginfo(i, f)
 
         return fragdict
+    
+    @classmethod
+    def from_smiles(cls, smiles, sanitize=True, addHs=True, **kwargs):
+        """Create a Universe from a SMILES string with rdkit
+
+        Parameters
+        ----------
+        smiles : str
+            SMILES string
+
+        sanitize : bool (optional, default True)
+            Toggle the sanitization of the molecule
+
+        addHs : bool (optional, default True)
+            Add all necessary hydrogens to the molecule
+
```

Comment:
How hard would it be to also generate coordinates?

Reply:
pretty easy

Reply:
`generate_coordinates=True` would be a fun kwarg then